pkg/ruler/scheduler.go

```diff
@@ -90,7 +96,7 @@ type userConfig struct {
 type groupFactory func(userID string, groupName string, rls []rules.Rule) (*group, error)
 
 type scheduler struct {
-	rulesAPI           RulesAPI
+	ruleStore          RuleStore
```

Comment:
For simplicity, might be best not to rename this type?  Feel free to do it in another PR.